elfi/model/extensions.py

```diff
@@ -16,6 +23,7 @@ def __init__(self, name=None):
         """
         self._name = name or self.__class__.__name__
 
+    @classmethod
```

Comment:
? Isn't the MA2 example a good example of how they can use this?

I think we're comparing apples and oranges here? :)